readme.md

```diff
@@ -48,6 +49,22 @@ commands.spawn_bundle(Camera2dBundle::default())
     });
 ```
 
+Clamped camera example
```

Comment:
Could you move this to a separate example in the examples folder? Having them as actual examples makes it easier for people to run them, and also makes it easier to maintain, since they are build tested in ci.